common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.util;
+
+import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+
+import org.slf4j.Logger;
+
+import tachyon.Constants;
+
+/**
+ * Utility methods to parse specific formats, print according to specific formats or transform among
```

Comment:
Is this file only about String formats? If so, the comments should mention String. Also, you may want to consider renaming the class to "StringFormatUtils" or something, since only "Format" may be confusing.